packages/firestore/test/integration/api/README.md

```diff
@@ -0,0 +1,4 @@
+All tests in this directory are also run against our (minified)
+firebase-firestore.js build (via 'yarn test:manual' under
+firebase-js-sdk/integration/firestore/) and therefore must not have any
+dependencies on src/ files.
```

Comment:
Add a note here indicating that tests run against the API but otherwise need internal hooks (i.e. can't be written purely against the API) should go in `test/integration/api_internal`?